app/src/main/java/com/sunasterisk/boringweather/ui/current/CurrentFragment.kt

```diff
@@ -0,0 +1,64 @@
+package com.sunasterisk.boringweather.ui.current
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.view.GravityCompat
+import androidx.core.widget.NestedScrollView
+import androidx.fragment.app.Fragment
+import com.sunasterisk.boringweather.R
+import com.sunasterisk.boringweather.util.scrollProgress
+import kotlinx.android.synthetic.main.fragment_current.*
+import kotlinx.android.synthetic.main.fragment_current.view.*
+
+class CurrentFragment private constructor() : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_current, container, false)
+    }
+
+    @SuppressLint("RestrictedApi")
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initView()
+    }
+
+    private fun initView() {
+        with(recyclerTodaySummaryWeather) {
+            adapter = SummaryWeatherAdapter("°C")
+        }
+        with(recyclerForecastSummaryWeather) {
+            adapter = SummaryWeatherAdapter("°C")
+        }
```

Comment:
hard code 
Không phải lúc nào cũng dùng run, let, with, ..., khi nào gọi nhiều lần trở lên thì mình mới cân nhắc dùng vì nó sẽ phức tạp đoạn code của mình lên 
```suggestion
            recyclerTodaySummaryWeather.adapter = SummaryWeatherAdapter(TempuraUnits.SOME_THING)
            recyclerForecastSummaryWeather.adapter = SummaryWeatherAdapter(TempuraUnits.SOME_THING)
```